layouts/dstu1_api.html

```diff
@@ -1,6 +1,6 @@
 <%= render 'head' %>
 
-<body class="api">
+<body class="api dstu1_api">
```

Comment:
Just use the dstu-specific class since that's what has representation in the css.